src/Microsoft.ML.CpuMath/FactorizationMachine/FactorizationMachineInterface.cs

```diff
@@ -0,0 +1,41 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.InteropServices;
+using System.Security;
+using Microsoft.ML.Internal.CpuMath.Core;
+
+namespace Microsoft.ML.Internal.CpuMath
+{
+    internal static unsafe partial class FieldAwareFactorizationMachineInterface
+    {
+        internal const string NativePath = "FactorizationMachineNative";
```

Comment:
Unfortunately, now that we've moved this code into CpuMath, this dependency is now backwards.

The `FactorizationMachineNative` assembly ships in the `Microsoft.ML` nuget package, which has a dependency on the `Microsoft.ML.CpuMath` nuget pacakge. However, this is now creating a runtime dependency from CpuMath back to an assembly shipping in the `Microsoft.ML` nuget package.

I think the simplest fix for this would be to remove the `FactorizationMachineNative` native assembly all together, and put its 2 functions into the CpuMathNative assembly, which ships in this same nuget package.